core/frontend-stubs/maven-frontend-stub/src/test/java/org/eclipse/sw360/antenna/frontend/stub/mojo/DependencyTest.java

```diff
@@ -62,6 +62,7 @@
             dependency("org.gradle", "gradle-base-services"),
             dependency("org.gradle", "gradle-core-api"),
             dependency("org.gradle", "gradle-logging"),
+    	    dependency("org.gradle", "gradle-base-services-groovy"),
```

Comment:
This (and all other `gradle-base-services-groovy` occurrences) should not be needed anymore (I've just recently removed them in the upgrade to Gradle 5.6).

Reply:
I explicitely had to add this due to a build problem 🤔 . The build told me that this dependency is missing.

Reply:
As CI passed for my PR, could you please double-check you've got a clean build?

Reply:
Whatever happened locally, now I do not need it. You were right, change pushed